sim/mage/frost/frost.go

```diff
@@ -23,25 +23,90 @@ func RegisterFrostMage() {
 	)
 }
 
+type FrostMage struct {
+	*mage.Mage
+	ClassBaseScaling float64
+
+	waterElemental *WaterElemental
+}
+
 func NewFrostMage(character *core.Character, options *proto.Player) *FrostMage {
 	frostOptions := options.GetFrostMage().Options
 
 	frostMage := &FrostMage{
-		Mage: mage.NewMage(character, options, frostOptions.ClassOptions),
+		Mage:             mage.NewMage(character, options, frostOptions.ClassOptions),
+		ClassBaseScaling: 937.330078125,
```

Comment:
use the util function for getting the base scaling `core.GetClassSpellScalingCoefficient`